### DISCUSSION TITLE
Show multiple slides simultaneously

### DISCUSSION THREAD

**Discussion by mrcury** [yifaneye/react-gallery-carousel#30] [Ideas]
Hi! Great component :) I was thinking it would be great to have an option to show (and swipe) more than one slide at a time, mainly to have different layouts for different screen sizes. A quick solution I found is splitting slides into "sub-slides", but that means reloading the component every time the device orientation changes.

**Comment by yifaneye** [comment]
Hi @mrcury !

Thank you for your appreciation! 😄

I also think "it would be great to have an option to show (and swipe) more than one slide at a time". For now, this component has only 1 slide at a time. 

Your quick solution works! For the scenario you mentioned, the easy solution is to set breakpoints that contain both screen widths for your devices' orientations to have the same number of "sub-slides". If this is not what you want, my tip for you will be to write an adaptor (wrapper) around this component and map the current item to the current index of slides you feed to the component. This way, you can achieve what you wanted. However, I have seen other components out there that can do this, e.g. 'slick'.
